### DISCUSSION TITLE
npm-proxy Remote storage url ?

### DISCUSSION THREAD

**Discussion by zhaozhuangwang** [verdaccio/verdaccio#3102] [Q&A]
centos7 
nexus  npm  proxy  

How to delegate?

This proxy does not find the dependency package, but contains it in the private library !

![image](https://user-images.githubusercontent.com/42265723/160984357-0432e484-5efb-4b4f-83f3-68150c004b0d.png)

The following is normal

nexus  npm  proxy Remote storage  http://registry.npm.taobao.org/

npm config set registry  centos7 nexus  url   

npm i

**Comment by juanpicado** [comment]
Hi, if this is an issue with Nexus I think you might need to ask in their forums, I doubt you can find any help here. If you find issue on verdaccio, you might need give away more information related with verdaccio. While is not a bug please let`s keep discussion here.

**Reply by zhaozhuangwang** [reply]
Thank you very much for your answer, because there is no problem in proxy other addresses in Nexus, but if you use Verdaccio proxy, an error will be reported.

![image](https://user-images.githubusercontent.com/42265723/161017111-3ca917ab-0238-4fa7-9b3f-8464ad39480d.png)


http://120.24.62.150:4873/

**Reply by juanpicado** [reply]
We can start with the error, which it is ? Then I can ask more questions.

**Reply by zhaozhuangwang** [reply]
Refer to the top for configuration

http://120.24.62.150:4873/
This is the address of a private library, I want to download TD-vuE-UI, this private library is not included, but the download process

npm ERR! notarget No matching version found for td-vue-ui@^1.2.10.

**Reply by juanpicado** [reply]
Ok I undersand that but that s a legitimate error, you need too look which configuration you have in verdaccio, it requires authetication? are you sending the token? have yo look into the logs, you need to look in your enviroment and you will find out.

I cannot help with Nexus, that  is something you need to look by yourself in their documentation.
